File: CodeChef/Self Learning/Basic Programming Concepts/Vaccine_Dates.cpp
// CodeChef - VDATES - Vaccine Dates
// https://www.codechef.com/LP1TO201/problems/VDATES

#include "bits/stdc++.h"
using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    int T;
    cin >> T;
    int D, L, R;
    while(T--){
        cin >> D >> L >> R;
        string ans = "Too Late";
        if(D <= R) {
            ans = "Take second dose now";
        }
        if(D < L) {
            ans = "Too Early";
        }
        cout << ans << endl;
    }

}

File: CodeChef/Self Learning/Basic Programming Concepts/Vaccine_Dates.py
# CodeChef - VDATES - Vaccine Dates
# https://www.codechef.com/LP1TO201/problems/VDATES

import sys
input = sys.stdin.readline

T = int(input())
for _ in range(T):
    D, L, R = map(int, input().split())
    ans = "Too Late"
    if D <= R:
        ans = "Take second dose now"
    if D < L:
        ans = "Too Early"
    print(ans)
    